internal/ingress/annotations/annotations_test.go

```diff
@@ -312,3 +313,44 @@ func TestCustomHTTPErrors(t *testing.T) {
 		}
 	}
 }
+
+func TestCustomResponseHeaders(t *testing.T) {
+	ec := NewAnnotationExtractor(mockCfg{})
+	ing := buildIngress()
+
+	fooAnns := []struct {
+		annotations map[string]string
+		headers     map[string]string
+	}{
+		{map[string]string{annotationCustomResponseHeaders: "Content-Type: application/json"}, map[string]string{"Content-Type": "application/json"}},
+		{map[string]string{annotationCustomResponseHeaders: `Content-Type: application/json
+		Accept: application/json
+		`}, map[string]string{"Content-Type": "application/json", "Accept": "application/json"}},
+		{map[string]string{annotationCustomResponseHeaders: `Content-Type application/json
+		Accept: application/json
+		`}, map[string]string{}},
+		{nil, map[string]string{}},
```

Comment:
I think we can add a case for when the same content is specified multiple times in a header.

Reply:
@tao12345666333 Let me take a look.

Reply:
>  I think we can add a case for when the same content is specified multiple times in a header.

@tao12345666333 This test uses map for the implementation, that doesn't allow duplicate keys. I will have to change the way this test uses data from map to a custom struct.